### DISCUSSION TITLE
배열을 어떻게 사용해야되는건지 모르겠네요

### DISCUSSION THREAD

**Discussion by shogun0331** [naverz/zepeto-studio-kor#168] [5. Q&A]
안녕하세요 
아래와 같이 게임오브젝트 배열을 사용하려 합니다.

 //Keys
    public Keys : GameObject[];

    Awake(){
        for(var i = 0; i< this.Keys.length; ++i){
          
        }}
}

"TypeError: Cannot read property 'length' of undefined"

정의 되지 않았다고 하는데.. 배열을 어떻게 사용 해야 되나요?

**Comment by x4cf** [comment]
이전 질문에 의심되는 문제를 답변하였습니다 확인해주세요 :D
